build-pipeline/dotnet-docker-linux-arm32v7-images.json

```diff
@@ -166,6 +185,9 @@
     "docker.commonRunArgs": {
       "value": "--rm -v $(docker.volumeName):/repo -w /repo --name $(docker.containerName)"
     },
+    "docker.runArgs": {
```

Comment:
This is not a good variable name.  The name is ambiguous with docker.commonRunArgs.  `docker.runImageBuilder` seems more appropriate.

Reply:
Updated.